### PR TITLE
fead: look and feel ajustments in signup

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Remove content promotion. (ES/EN)
- Eliminate unnecessary classes (link - green). (ES/EN)
- Add a modifier class for the button "button--round" 
- Add link Privacy Policies in the footer. (ES/EN)
- Add the Show / hide content in the password field (ES / EN)
- Remove the unused Keys "promotion_code_help_url" and "promotion_code_reminder".
- I made the comparison with Login. (ES/EN)

ajustments-signup

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -67,28 +69,26 @@
   "reports_title": "Doppler | Reports",
   "signup": {
     "activate_account_instructions": "* By clicking on the button from the Email, you will activate your account and you will be ready to enjoy all the benefits of Doppler.",
-    "button_signup": "Sign Up",
+    "button_signup": "create your free account",
     "check_inbox": "Check your inbox. You have an email!",
     "check_inbox_icon_description": "Check your inbox",
-    "copyright": "© {year} Doppler LLC. All rights reserved.",
+    "copyright": "© {year} Doppler LLC. All rights reserved. <a href=\"https://www.fromdoppler.com/en/legal/privacy-policy\">Privacy and Legal Policy.</a>",
```
Comment (andresmoschini) [inline src/i18n/en.json]: Maybe:

```suggestion
    "copyright": "© {year} Doppler LLC. All rights reserved. <a target=\"_blanc\" href=\"https://www.fromdoppler.com/en/legal/privacy-policy\">Privacy and Legal Policy.</a>",
```

Take it into account in other links.

**Comment by andresmoschini** [comment]
👍

File: src/i18n/en.json (hunk 1 above)
Comment (GusBaamonde) [reply src/i18n/en.json]: Ok I add it!

**Comment by andresmoschini** [comment]
🚀

**Comment by andresmoschini** [comment]
:shipit:

**Pull request merged**